abdul-moize/django_ecommerce
# Feat: Templates Login & Register

This PR adds Templates of  Login & Register to main branch.

To Test this PR follow these steps:
1. Pull this branch and run: `pip install -r requirements.txt`
2. run `python manage.py migrate && python mange.py runserver`
3. Open `127.0.0.1:8000/users/login/` in browser
For now you can log into the system or register a user
There is not a homepage yet. But you can test that log in works by loggin in as user then opening `127.0.0.1:8000/admin/` you will see it does not require login.
Please let me know if there are any comments, concerns or issues.
